tests/test_interface.py

```diff
@@ -335,7 +335,7 @@ def test_interface_add_garbage(self):
 
     @mock.patch.object(Payment, 'deserialize')
     @mock.patch.object(Payment, 'serialize', return_value=CC_RETURN)
-    @mock.patch.object(app_db, 'session', autospec=True)
+    @mock.patch.object(app_db, 'session')
```

Comment:
Probably for the best to remove `autospec=True`.  It can be a little more trouble than it's worth in mocking actually.  I know that we have it in a bunch of places in our tests, but it's permissible for now.